sui-execution/src/latest.rs

```diff
@@ -187,4 +189,25 @@ impl<'m> verifier::Verifier for Verifier<'m> {
     ) -> SuiResult<()> {
         run_metered_move_bytecode_verifier(modules, &self.config, &mut self.meter, self.metrics)
     }
+
+    fn meter_compiled_modules_with_overrides(
+        &mut self,
+        modules: &[CompiledModule],
+        _protocol_config: &ProtocolConfig,
+        config_overrides: &VerifierOverrides,
+    ) -> SuiResult<VerifierMeteredValues> {
+        let max_per_fun_meter_current = self.config.max_per_fun_meter_units;
+        let max_per_mod_meter_current = self.config.max_per_mod_meter_units;
+        self.config.max_per_fun_meter_units = config_overrides.max_per_fun_meter_units;
+        self.config.max_per_mod_meter_units = config_overrides.max_per_mod_meter_units;
```

Comment:
Don't you need to set these back to how they were, or, even better, seeing as you are passing a reference to the `config` in the call below, how about taking a copy of the `config` and editing that instead?

Reply:
yes will do and good point!